ci/infra/testrunner/vars.yaml

```diff
@@ -37,11 +37,11 @@ lb:
   cpu: 4
 
 master:
-  count: 3
+  count: 4 # Amount of masters to provision
```

Comment:
I think that changing the defaults  only for one tests is not the best option. Maybe simplifying the test to simply remove  one of the master without previously joining it will solve all these issues.

Reply:
removed this